openslides_backend/action/actions/meeting/import_.py

```diff
@@ -192,32 +192,24 @@ def replace_field_ids(
             fqids_str = RECOMMENDATION_EXTENSION_REFERENCE_IDS_PATTERN.findall(
                 entry[field]
             )
-            # split the entry in different parts.
             entry_str = entry[field]
             entry_list = []
             for fqid in fqids_str:
                 search_str = "[" + fqid + "]"
                 idx = entry_str.find(search_str)
                 entry_list.append(entry_str[:idx])
-                entry_list.append(entry_str[idx : idx + len(search_str)])
-                entry_str = entry_str[idx + len(search_str) :]
+                col, id_ = fqid.split(KEYSEPARATOR)
+                replace_str = (
+                    "["
+                    + collection
```

Comment:
Please use `col` here - I think currently only `motion` is allowed, but this might change; also it's confusing to mix these two variables

Reply:
Yes, I should use` col,` here. I will change it.